Controllers/Reviews_controller.js

```diff
@@ -21,5 +22,20 @@ function getReviews(req, res, next) {
     })
 }
 
+function updateVotes(req, res, next) {
+    const {review_id} = req.params
+    const votesToAdd = req.body["inc_votes"]
+
+    updateVotesModel(review_id, votesToAdd).then((updatedObj) => {
+        if (updatedObj.length === 0){
+            return checkIfIdExists(review_id)
+        }
+        res.status(201).send({updatedVotes: updatedObj[0]})
```

Comment:
As we are sending back the entire review, not just the updatedVotes key can we use a more appropriate key on the response object

Reply:
Will do!